stock_warehouse_flow_packaging_type/README.rst

```diff
@@ -0,0 +1,81 @@
+===================================
+Stock Warehouse Flow Packaging Type
```

Comment:
Maybe rename to stock_warehouse_flow_product_packaging ?
As product_packaging_type is now product_packaging_level in v16 https://github.com/OCA/product-attribute/pull/1215